### DISCUSSION TITLE
Split the examples into individual repos!

### DISCUSSION THREAD

**Discussion by babakfp** [supabase/supabase#21029] [General]
Hi
I was cloning this repo to access one of the examples but it seems like the size of the repo is huge and I canceled the cloning after 50mb of download! So, my suggestion is to split the examples into individual repos.

**Comment by charislam** [comment]
Thanks for the feedback! Maybe a [sparse checkout](https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/) would help you right now?
